src/components/app-container.js

```diff
@@ -86,7 +86,7 @@ function Container({ location }) {
                             <Route path="/recently-viewed">
                                 <RecentlyViewedPage data={spaces.data[0]} />
                             </Route>
-                            <Route path="/" render={() => <HomePage history={history} />} />
+                            <Route path="/" render={() => <HomePage history={history} data={spaces.data} />} />
```

Comment:
Will have to change this after you or I merge our PR. In my PR, I have a new variable that combines the json data (spaces.data) with the data from localStorage (customSpaces.data). Check my PR in app-container.